packages/components/src/components/Grid/index.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import css from '@styled-system/css';
+
+const fontSize = 1; // rem = 16px
+const lineHeight = fontSize * 1.5;
+
+export const Grid = styled.div<{ columnGap?: number; rowGap?: number }>(
+  ({ columnGap, rowGap }) =>
+    css({
+      display: 'grid',
+      gridTemplateColumns: 'repeat(12, 1fr)', // always 12 columns
+      gridColumnGap:
+        (typeof columnGap !== 'undefined' ? columnGap : lineHeight * 2) + 'rem',
+      gridRowGap: (typeof rowGap !== 'undefined' ? rowGap : lineHeight) + 'rem',
+    })
+);
+
+// todo: end and span cant be together
+// valid combinations are
```

Comment:
```suggestion
```

Reply:
If we wanna change I think its nice to have the comments

Reply:
Reading the updated type, will let you know what's possible and what not